### ISSUE TITLE
COMP File storage error

### ISSUE THREAD

**Issue by hdoupe** [compute-tooling/compute-studio#188]
sim: 
- https://compmodels.org/PSLmodels/Tax-Brain/42034/
- https://compmodels.org/PSLmodels/Tax-Brain/42035/

traceback:
```
Traceback (most recent call last):
  File "/home/distributed/api/celery_app/__init__.py", line 79, in f
    outputs = s3like.write_to_s3like(task_id, outputs)
  File "/opt/conda/lib/python3.7/site-packages/s3like/__init__.py", line 139, in write_to_s3like
    buff, OBJ_STORAGE_BUCKET, ziplocation, ExtraArgs={"ACL": "public-read"}
  File "/opt/conda/lib/python3.7/site-packages/boto3/s3/inject.py", line 539, in upload_fileobj
    return future.result()
  File "/opt/conda/lib/python3.7/site-packages/s3transfer/futures.py", line 106, in result
    return self._coordinator.result()
  File "/opt/conda/lib/python3.7/site-packages/s3transfer/futures.py", line 265, in result
    raise self._exception
  File "/opt/conda/lib/python3.7/site-packages/s3transfer/tasks.py", line 126, in __call__
    return self._execute_main(kwargs)
  File "/opt/conda/lib/python3.7/site-packages/s3transfer/tasks.py", line 150, in _execute_main
    return_value = self._main(**kwargs)
  File "/opt/conda/lib/python3.7/site-packages/s3transfer/upload.py", line 692, in _main
    client.put_object(Bucket=bucket, Key=key, Body=body, **extra_args)
  File "/opt/conda/lib/python3.7/site-packages/botocore/client.py", line 357, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/opt/conda/lib/python3.7/site-packages/botocore/client.py", line 661, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.exceptions.ClientError: An error occurred (ServiceUnavailable) when calling the PutObject operation (reached max retries: 4): Service is unavailable at this time

 ```

**Comment by hdoupe** [comment]
Digital Ocean's file storage service has been having issues today. This is what caused the issue. I've been looking into swapping over to Google's file storage system since we now use them for compute. Hopefully, that will be more reliable.

**Comment by hdoupe** [comment]
Compute-Studio's cloud storage client `cs-storage` has transitioned to Google Cloud Storage. This will go into effect when projects are all up to date with the changes specified in https://github.com/compute-tooling/compute-studio-kit/issues/4

**Comment by hdoupe** [comment]
#188 was resolved by #214. C/S now uses Google Cloud Storage for storing files. Hopefully, this will be more reliable than DO's Spaces.
